src/components/Footer.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PureComponent from 'react-pure-render/component';
+
+const styles = {
```

Comment:
Can you define `styles` under the component definition? Since the component definition is more important. So, it should be the first thing that someone who opens this file should see. And you don't need to worry about using a variable before declaring it, because in ES6 `const` are hoisted to the top of the block they are defined in.